src/components/views/mainPageView/CityCard.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  CardMedia,
+  Typography,
+} from '@material-ui/core'
+import { Link } from 'react-router-dom'
+import { makeStyles } from '@material-ui/core/styles'
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: 400,
+  },
+  media: {
+    height: 140,
+  },
+  noUnderLine: {
+    textDecoration: 'none',
+    '&:hover': {
+      textDecoration: 'none',
+    },
+  },
+}))
+
+export const CityCard = ({ url, city, cityDesc }) => {
+  const classes = useStyles()
+  return (
+    <Link
+      to={{ pathname: `/hotels/${city}`, state: { city: city } }}
+      className={classes.noUnderLine}
+    >
+      <Card className={classes.root}>
+        <CardActionArea>
+          <CardMedia className={classes.media} image={url} title={city} />
```

Comment:
add some kidn of hover to it,  lets say we want to make image wider inside image box (by 2 or 3px)
![image](https://user-images.githubusercontent.com/38506213/112025197-3b56bd80-8b35-11eb-8859-f2bd650246a7.png)